src/franka_plan_runner.cc

```diff
@@ -604,6 +607,13 @@ franka::JointPositions FrankaPlanRunner::JointPositionCallback(
   Eigen::VectorXd current_conf_franka =
       utils::v_to_e(ArrayToVector(cannonical_robot_state.q_d));
 
+  if (comm_interface_->IsSimulatingControlException()) {
+    dexai::log()->warn("Simulating control exception!");
+    RecoverFromControlException();
+    franka::JointPositions output_current = robot_state.q;
+    return franka::MotionFinished(output_current);
+  }
```

Comment:
this raises a control exception upon request via LCM? `output_current` is not current values in amperes but positions in radians

Reply:
> this raises a control exception upon request via LCM?

Yes that is the point of this PR